packages/relay/src/lib/eth.ts

```diff
@@ -119,16 +132,28 @@ export class EthImpl implements Eth {
 
     //contractTransactionGas is in tinycents * 1000, so the final multiplier is truncated by 3 zeroes for
     // the conversion to weibars
-    return Math.ceil(contractTransactionGas / exchangeRates.currentRate.cents *
-      exchangeRates.currentRate.hbars * 10_000_000);
+    return Math.ceil(
+      (contractTransactionGas / exchangeRates.currentRate.cents) *
+        exchangeRates.currentRate.hbars *
+        10_000_000
+    );
   }
 
   /**
    * Gets the most recent block number.
    */
-  async blockNumber() {
+  async blockNumber(): Promise<number> {
```

Comment:
```suggestion
  async blockNumber(): Promise<number | null> {
```
Will have to update it in `relay/src/index.ts` also

Reply:
done